### PR TITLE
Hide gridfield page count when there's no title

### PR DESCRIPTION
**Pull request by phalkunz**
Issue: https://github.com/silverstripe/silverstripe-admin/issues/256

### CODE REVIEW

File: src/Forms/GridField/GridFieldPageCount.php [hunk 1]
```diff
@@ -65,6 +74,10 @@ protected function getPaginator($gridField)
      */
     public function getHTMLFragments($gridField)
     {
+        if($this->config()->get('hidden_when_no_title') && !$gridField->Title()) {
```
Comment (robbieaverill) [inline src/Forms/GridField/GridFieldPageCount.php]: You'll need to fix this line for PSR-2 linting for it to pass the Travis builds

**Comment by tractorcow** [comment]
@phalkunz discussed possible better solution for this over on the parent story (see https://github.com/silverstripe/silverstripe-admin/issues/256#issuecomment-359282037)

**Comment by tractorcow** [comment]
Fix without extra api: https://github.com/silverstripe/silverstripe-admin/pull/403

**Comment by tractorcow** [comment]
Closing in favour of https://github.com/silverstripe/silverstripe-admin/pull/403

**Pull request closed without merging**